### ISSUE TITLE
Cannot craft supply Camp or Ship

### ISSUE THREAD

**Issue by eleshna** [ldtteam/minecolonies#3232]
### Minecolonies version
Minecolonies:
minecolonies-1.12.2-0.10.116-ALPHA-universal
minecolonies-1.12.2-0.10.109-RELEASE-universal
Forge : 
14.23.5.2768
Mods:
Minecolonies

### Expected behavior
Able to craft Supply Camp or Supply Ship
### Actual behaviour
If put boats or chest as (described in the wiki) into the workbench theres nothing in the output of the workbench.

If you get them with creative mode you can place them and everything works as expected.

### Steps to reproduce the problem
1. Install forge Client
2. put minecolonies in mod folder
3. start singelplayer world
4. craft supply camp or ship.

**Comment by Raycoms** [comment]
Works for me, is there any other mod installed?

**Comment by eleshna** [comment]
Only minecolonies is installed.

I have this Problem since Spring last year or sth like that.

**Comment by marchermans** [comment]
Send some screenshots please.

**Comment by eleshna** [comment]
Ok I solved it. 
The reason was, I had Minecraft normally installed and started to play minecolonies with the Modpack available on twitch. 
Somehow, the original Minecraft and Twich Minecraft don't get along together if I wanted to play minecolonies on the original minecraft(I hope you're able to follow me).
I uninstalled every single version from my pc an reinstalled Minecraft. 

I am very sorry for the inconvenience.
